dracut/modules.d/99kiwi-lib/kiwi-partitions-lib.sh

```diff
@@ -192,7 +192,8 @@ function get_partition_node_name {
     local part
     udev_pending
     for partnode in $(
-        lsblk -p -l -o NAME,TYPE,START -x START "${disk}" |\
+        { lsblk -p -l -o NAME,TYPE,START -x START "${disk}" 2>/dev/null ||\
```

Comment:
Can we add a comment to the method providing a bit of a reason for the two stage concept ?

Reply:
Done.